.editorconfig

```diff
@@ -8,8 +8,12 @@ trim_trailing_whitespace = true
 [*.md]
 indent_size = 2
 indent_style = space
-max_line_length = 100  # Please keep this in sync with bin/lesson_check.py!
-trim_trailing_whitespace = false  # keep trailing spaces in markdown - 2+ spaces are translated to a hard break (<br/>)
+
+max_line_length = 100
+# Please keep this in sync with bin/lesson_check.py!
+
+trim_trailing_whitespace = false
+# keep trailing spaces in markdown - 2+ spaces are translated to a hard break (<br/>)
```

Comment:
I think having the comments before the options are set (i.e., above rather than below) are clearer at signalling the reader what's coming and maybe safer that people would read it if changing it. But that may be a habit I've got.

Reply:
Thanks @dpshelio - I think you're right. I'll make that adjustment.